lib/controller.rb

```diff
@@ -83,15 +83,16 @@ class SlowFood < Sinatra::Base
     redirect '/auth/login'
   end
 
+  get '/menu' do
+    @menu = Menu.first
+    @dishes = Dish.all
```

Comment:
Creating the two routes was an error. Single route only exist and has also been refactored as recommended